src/CMakeLists.txt

```diff
@@ -9,10 +9,35 @@ set(SRC_FILES
 )
 
 add_library(tilt STATIC ${SRC_FILES})
-add_dependencies(tilt easy-jit-core)
 
-target_compile_options(tilt PUBLIC "SHELL:-Xclang -load" "SHELL:-Xclang ${easyjit_lib}")
-target_link_libraries(tilt EasyJitRuntime ${llvm_libs})
-
-target_include_directories(tilt PUBLIC ${LLVM_INCLUDE_DIRS} ../include ../third_party/easy_jit/include)
+target_link_libraries(tilt ${llvm_libs})
+target_include_directories(tilt PUBLIC ${LLVM_INCLUDE_DIRS} ../include)
 target_compile_options(tilt PRIVATE -Wall -Wextra -pedantic -Werror -Wno-unused-parameter)
+
+# Config for JIT
+set(NO_EASYJIT true)
+if(${NO_EASYJIT})
+    add_compile_definitions(NO_EASYJIT)
+endif()
+
+# Load required files
+file(STRINGS ../include/tilt/base/ctype.h VINSTR_TYPE NEWLINE_CONSUME)
+file(STRINGS ../include/tilt/pass/codegen/vinstr.h VINSTR_HEADER NEWLINE_CONSUME)
+file(STRINGS ../src/pass/codegen/vinstr.cpp VINSTR_SRC NEWLINE_CONSUME)
+
+# Replace preprocessors macros
+string(REGEX REPLACE "(^|\n)(#[^\n]*)(\n|$)" "" FILTERED_VINSTR_TYPE ${VINSTR_TYPE})
+string(REGEX REPLACE "(^|\n)(#[^\n]*)(\n|$)" "" FILTERED_VINSTR_HEADER ${VINSTR_HEADER})
+string(REGEX REPLACE "(^|\n)(#[^\n]*)(\n|$)" "" FILTERED_VINSTR_SRC ${VINSTR_SRC})
+
+# Create code content
+file(WRITE /tmp/vinstr.cpp "${FILTERED_VINSTR_TYPE}\n${FILTERED_VINSTR_HEADER}\n${FILTERED_VINSTR_SRC}")
```

Comment:
does it have to be an absolute path like `/tmp/vinstr.ll`?
We don't want to assume that `/tmp/` exists. you can just write to and read from the current directory where user is building the code?